telegram_handler.rb

```diff
@@ -3,27 +3,42 @@
 require 'tempfile'
 require 'gruff'
 
+
 module Mood
+
   class TelegramHandler
 
+    def self.customAnswers(_chatid)
```

Comment:
The Ruby convention is to use underscores to separate words in a multiword method or variable name.

Also, I would refactor "answers" into "moods_keyboard" to make it easier to comprehend.

Reply:
I'll make the changes but know that I took the underscore convention from here: 
https://www.techotopia.com/index.php/Ruby_Variable_Scope